cmd/juju/bundle/exportbundle.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package bundle
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/gnuflag"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/errors"
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewexportbundleCommand returns a fully constructed exportbundle command.
+func NewexportbundleCommand() cmd.Command {
+	return modelcmd.Wrap(&exportBundleCommand{}, modelcmd.WrapSkipModelFlags)
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out cmd.Output
```

Comment:
We don't want a cmd.Output because bundles are never valid in any other format.

Reply:
What we do probably want is a -f (--filename) option to write the bundle to a file.
Should default to ctx.Stdout.